src/gf_ring.cpp

```diff
@@ -59,25 +59,25 @@ void RingModN<uint32_t>::sub_two_bufs(
     uint32_t* res,
     size_t len)
 {
-    simd::u32::sub_two_bufs(bufa, bufb, res, len, this->_card);
+    simd::sub_two_bufs_u32(bufa, bufb, res, len, this->_card);
 }
 
 template <>
 void RingModN<uint16_t>::add(int n, uint16_t* x_u16, uint16_t* y_u16)
 {
     int half = n / 2;
```

Comment:
`half` in all of these functions can be declared const.

Reply:
it's done